1-Draft/RFC00XX-Update-Error-View.md

```diff
@@ -0,0 +1,182 @@
+---
+RFC: Update $ErrorView with Single String Error Messages
+Author: Jason Helmick
+Status: Draft
+Version: 0.0.0
+Area: PowerShell
+Comments Due: 08/31/2019
+Plan to implement: Yes
+---
+
+# Update $ErrorView with Single String Error Messages
+
+When an error occurs in PowerShell, the customers on-screen error message experience currently
+provides a level of detail that obscures the inner exception message from being read by
+new or occasional PowerShell users.
+To improve both comprehension and the troubleshooting experience, a single string of text
+containing the error will be displayed.  A cmdlet will provide the full error details when desired.
+
+## Motivation
+
+The on-screen experience, when receiving an error message,
+is controlled through the views NormalView (the default) and CategoryView. These are user selectable though the preference variable $ErrorView.
+The addition of a “ConciseView” as default that contains only specific error
```

Comment:
I prefer the recommendation to call it `SimpleView` as concise is more specific and implies some intelligence to determine what error should be shown (in the case of multiple inner exceptions, for example) which isn't going to be the case.

Reply:
Yes. Think of non-native speakers of English. Chances that they will understand "simple" ... quite good. "Concise"  ... not very good. Prefer the plainer word.